### ISSUE TITLE
Question about the loss_ort

### ISSUE THREAD

**Issue by ShiQingHongYa** [mdswyz/DMD#2]
When running the code, there is wrong for "cosine_similarity_s_c_l, cosine_similarity_s_c_v, cosine_similarity_s_c_a" in DMD.py. The input (output['s_l'], output['c_l']) of the cosine_similarity_s_c_l loss is [46,16,50], which is 3 dimensions, but the loss requests 2 dimensions.

**Comment by ShiQingHongYa** [comment]
Thanks for your answer. I have changed the output['s_l] when computing loss_ort, but the results (85.) are not consistent with the paper (86.)on CMU-MOSI dataset.

**Comment by mdswyz** [comment]
You can try different seeds to tune results. Our default setting can obtain 87.8  acc2 in mosi dataset (which is higher than our results reported in the paper) if your environment is the same as ours (pytorch1.9.0 and python3.8). We also provide the pre-trained model with 87.8 acc2, 87.7 f1, and 46.4acc7.

**Comment by mdswyz** [comment]
You also can change to output['s_l'].contigouuous.view(labels.size(0),-1) to calculate cosine_similarity.

**Comment by ShiQingHongYa** [comment]
> You can try different seeds to tune results. Our default setting can obtain 87.8 acc2 in mosi dataset (which is higher than our results reported in the paper) if your environment is the same as ours (pytorch1.9.0 and python3.8). We also provide the pre-trained model with 87.8 acc2, 87.7 f1, and 46.4acc7.

thank you very much! I will try different seeds.
